features/manuals_publisher.feature

```diff
@@ -0,0 +1,8 @@
+@app-manuals-publisher
+Feature: Manuals Publisher
+
+  @local-network
+  Scenario: Healthcheck
+    Given I am testing "manuals publisher" internally
```

Comment:
I think this needs the hyphen (`manuals-publisher`), otherwise [Plek won't be able to find it](https://github.com/alphagov/smokey/blob/f5ff5b39b060fa7b933453c8b9644ec871c04e6e/features%2Fsupport%2Fbase_urls.rb#L24).

Reply:
Fixed